src/proto/data_sync.proto

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto2";
+
+// datasync is a tool to sync data from online to offline
+// need java package cus SyncTool is a java class
+package openmldb.datasync;
+option java_package = "com._4paradigm.openmldb.proto";
+option java_outer_classname = "DataSync";
+option cc_generic_services = true;
+
+enum SyncType {
+    SNAPSHOT = 0;
```

Comment:
the name style of enum refer https://google.github.io/styleguide/cppguide.html#Enumerator_Names